### DISCUSSION TITLE
External link preview and/or link-type note

### DISCUSSION THREAD

**Discussion by kjgcoop** [TriliumNext/Trilium#4802] [Ideas] [answered]
Is there any notion to add functionality to fetch page information when an external URL is added? Like I paste a URL and Trilium retreives its human name and maybe a thumbnail? Even if it was a new note type such that each note could only have the one link.

Below is what it looks like in Linkwarden:
![Screenshot_20240611_141129_Firefox](https://github.com/zadam/trilium/assets/51257881/4f82f860-1fe0-4b6c-9cd6-fc3204758933)

I realize this is nontrivial and far from vital, I just think it might be cool. It would be nice keep all my notes in Trilium.

Is there any chance to may come to be?

Thank you!

**Comment by meichthys** [comment, marked as answer]
There's a chance, but probably not on this repo as [Trilium is in maintenance mode](https://github.com/zadam/trilium/issues/4620). Perhaps you could fine a feature request over on [TriliumNext](https://github.com/TriliumNext/Notes).
